pkg/kubelet/apis/deviceplugin/v1beta1/api.proto

```diff
@@ -74,15 +74,19 @@ message ListAndWatchResponse {
 }
 
 message TopologyInfo {
-    	int64 socket = 1;
+    	NUMANode node = 1;
+}
+
+message NUMANode {
+    int64 id = 1;
```

Comment:
This should probably be `ID` instead of `id` to match the convention elsewhere.

Reply:
Updated, thanks!